### PR TITLE
Remove fapolicyd prevent home folder access from the RHEL 9 STIG

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

- Remove fapolicyd_prevent_home_folder_access from the RHEL 9 STIG
- Remove the harmful text from fapolicyd_prevent_home_folder_access

#### Rationale
Remove bad advice from the project.

### CODE REVIEW

File: linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml [hunk 1]
```diff
@@ -21,21 +21,11 @@ references:
...
-    Add or edit the following lines in /etc/fapolicyd/rules.d/90-deny-home.
-
-    deny_audit perm=chmod path=/home all : all
-
-    Note: That fapolicyd must have correctly configured rules. All configurations will be based on the actual system setup and organizational polices and practices.
-
-srg_requirement: '{{{ full_name }}} must limit the ability of non-privileged users to grant other users direct access to the contents of their home directories/folders.'
+warnings:
+    - general: |-
+          This rule is deprecated and there is no replacement at this time.
```
Comment (jan-cerny) [inline linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml:31]: I need some explanation - there is no reason mentioned here or in the commit messages or in the PR description. You wrote that it's a "bad advice" but I'm curious why.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml (hunk 1 above)
Comment (Mab879) [reply linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml:31]: If you apply this rule fapolicyd will not start.

Comment (jan-cerny) [reply linux_os/guide/services/fapolicyd/fapolicyd_prevent_home_folder_access/rule.yml:31]: @Mab879 so please add it to the warning text

**Review by jan-cerny — APPROVED** [review]
I think that you could make it even more transparent

**Pull request merged**